shell/platform/windows/direct_manipulation.h

```diff
@@ -22,6 +22,7 @@ class DirectManipulationEventHandler;
 class DirectManipulationOwner {
  public:
   explicit DirectManipulationOwner(Window* window);
+  virtual ~DirectManipulationOwner() = default;
```

Comment:
Consider implementing the rule of C++ rule of 5. https://stackoverflow.com/questions/26039907/does-rule-of-three-five-apply-to-inheritance-and-virtual-destructors

Reply:
Thanks for the resource! I added a `FML_DISALLOW_COPY_AND_ASSIGN`. It looks like the engine's style is to skip the move constructor in this mocking case, so I skipped that.